workspaces/acs/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "20.17.0"
+    "node": "20.19.1"
```

Comment:
```suggestion
    "node": "20.x"
```

@maknop, would you be open to pinning your engines to the major line here?

Reply:
^ @maknop I think that should be fine. Let me know if you feel different.